doc/advanced_configuration.rst

```diff
@@ -336,10 +345,12 @@ saved in the directory we specified in the *toctree* option of the
 The files we are including are from the ``backreferences_dir``
 configuration option setup for Sphinx-Gallery.
 
+.. _code_tesiii:
```

Comment:
This reference is not used anywhere is it?

Reply:
I don't see it added anywhere, @Titan-C what is this for?

Reply:
Since I added in `doc/_templates/module.rst` also the instruction to document classes I wanted to seed how this was affected on the line numbers. This reference was for personal use when reloading the HTML should have not been committed. 
As a side issue. It appears that the readthedocs theme on a python3 build(at least mine) cannot align the line numbers with the code lines. Other sphinx themes and our py2 build on CircleCI do display correctly.